scripts/generate/templates/SUBSCRIPTION_MANAGEMENT/vanilla.js.template

```diff
@@ -193,40 +214,45 @@ function Remove(root, api) {
 
 // 'Edit' mode should modify an existing selling plan.
 // Changes should affect other products that have this plan applied.
+// [Shopify admin renders this mode inside an app overlay container]
 function Edit(root, api) {
   const data = api.data;
   const sessionToken = api.sessionToken;
-  const {close, done, setPrimaryAction, setSecondaryAction} = api.container;
+  const {close, done} = api.container;
 
-  setPrimaryAction({
-    content: 'Edit plan',
-    onAction: async () => {
-      // Get a fresh session token before every call to your app server.
+  const primaryButton = root.createComponent(Button, {
+    title: 'Edit plan',
+    primary: true,
+    onClick: async () => {
```

Comment:
No longer needs to be async?

Reply:
Looks like we still need it for `await getSessionToken()` 👌